controllers/storagecluster/storageclasses.go

```diff
@@ -371,6 +371,8 @@ func newEncryptedCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageC
 	encryptedStorageClassConfig := newCephBlockPoolStorageClassConfiguration(initData)
 	encryptedStorageClassConfig.storageClass.ObjectMeta.Name = generateNameForEncryptedCephBlockPoolSC(initData)
 	encryptedStorageClassConfig.storageClass.Parameters["encrypted"] = "true"
+	encryptedStorageClassConfig.storageClass.Annotations["cdi.kubevirt.io/clone-strategy"] = "copy" // https://issues.redhat.com/browse/RHSTOR-4461
```

Comment:
```suggestion
	encryptedStorageClassConfig.storageClass.Annotations["cdi.kubevirt.io/clone-strategy"] = "copy"
```
Better to add a comment about what this annotation does instead of pointing it to a Jira issue.